Mobile Buy SDK/Mobile Buy SDK/Data/BUYClient+Checkout.h

```diff
@@ -220,6 +220,13 @@ typedef void (^BUYDataGiftCardBlock)(BUYGiftCard * _Nullable giftCard, NSError *
  */
 - (NSOperation *)removeGiftCard:(BUYGiftCard *)giftCard fromCheckout:(BUYCheckout *)checkout completion:(BUYDataCheckoutBlock)block;
 
+/**
+ *	Removes the discount from the checkout.
+ *
+ *  @param checkout An existing BUYCheckout on Shopify
+ */
+- (void)removeDiscountFromCheckout:(BUYCheckout *)checkout;
```

Comment:
I don't see the need for a new public function for this.  On android you can do this by calling checkout.setDiscountCode(null) and doing an update call